src/sentry/rules/conditions/event_frequency.py

```diff
@@ -63,7 +63,7 @@ class EventFrequencyForm(forms.Form):
     )
     value = forms.IntegerField(widget=forms.TextInput())
     comparisonType = forms.ChoiceField(
-        choices=list(sorted(comparison_types.items(), key=lambda item: item[1])),
+        choices=list(sorted(comparison_types.items(), key=lambda item: str(item[1]))),
```

Comment:
`str(...)` call seems suspicious here -- what diagnostic was this fixing?

`list(sorted(...))` is also suspect -- sorted already returns a list

also dicts are ordered in python 3.6+ so we probably don't even need to sort

Reply:
> also dicts are ordered in python 3.6+ so we probably don't even need to sort

thought so too, I'll simplify this!